GiniVision/Classes/GiniPDFDocument.swift

```diff
@@ -56,6 +57,42 @@ final public class GiniPDFDocument: NSObject, GiniVisionDocument {
         }
     }
     
+    fileprivate func renderFirstPage(fromPdf pdf:CGPDFDocument) -> UIImage? {
+        var pdfImage:UIImage?
+        let pdfDoc = pdf
+        
+        if let pdfPage:CGPDFPage = pdfDoc.page(at: 1) {
```

Comment:
I hope this 1 is safe and wont crash for PDF's with 0 zero pages (if that's possible)...

Reply:
It would be `nil` and it won't continue, the only thing is that the pdfImage would be `nil`